libs/ledger-live-common/src/api/Ethereum.ts

```diff
@@ -302,7 +302,7 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
         return data
           .map((m: any) => {
             if (!m || typeof m !== "object") return;
-            const { spender, value } = m;
+            const { spender, count: value } = m;
```

Comment:
Not since v4... 
Not covered either, I'm just fixing it here because I was testing if I could remove some dead code from compound and realized no approval was ever fetched. That's the kind of things that would be catch by the integration tests I guess as we would see the missing approvals.
Not covering it here because our only use case for this method is compound which is (100% brokend and) being actively removed. 

FYI @hedi-edelbloute